CRM/Contribute/Form/Contribution/Main.php

```diff
@@ -1079,19 +1081,7 @@ public function submit($params) {
     }
 
     $params['separate_amount'] = $params['amount'];
-    $memFee = NULL;
-    if (!empty($params['selectMembership'])) {
-      if (empty($this->_membershipTypeValues)) {
-        $this->_membershipTypeValues = CRM_Member_BAO_Membership::buildMembershipTypeValues($this,
-          (array) $params['selectMembership']
-        );
-      }
-      $membershipTypeValues = $this->_membershipTypeValues[$params['selectMembership']];
-      $memFee = $membershipTypeValues['minimum_fee'];
```

Comment:
$memFee is replaced further down by this->hasSeparateMembershipPaymentAmount($params))